js/exmo.js

```diff
@@ -1956,4 +1948,9 @@ module.exports = class exmo extends Exchange {
             }
         }
     }
+
+    safeTickSize (object, key) {
+        const precisionDigitsString = this.safeString (object, key);
+        return this.parseNumber (this.parsePrecision (precisionDigitsString));
+    }
```

Comment:
We can't use the name `object` here: https://stackoverflow.com/a/10044533/5055465

```suggestion
    safeTickSize (info, key) {
        const precisionDigitsString = this.safeString (info, key);
        return this.parseNumber (this.parsePrecision (precisionDigitsString));
    }
```